ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -3663,3 +3663,53 @@ def test_numpy_ndarray_round(
         method_flags=method_flags,
         on_device=on_device,
     )
+    
+#var
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="var",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        valid_axis=True,
+    ),
+    keepdims=st.booleans(),
+    where=np_frontend_helpers.where(),
+    dtype=helpers.get_dtypes(kind="integer"),
```

Comment:
You can take a look at the test of the frontend `np.var` - there's a helper function used there `_statistical_dtype_values` which you can use to generate the values. It makes sure that the inputs have reasonable bounds and it will also take care of your `ddof` parameter

Reply:
to use it I have to import it, is it allowed to import new modules?